### PR TITLE
test: [M3-7557] - remove console logs from e2e tests

### PR DESCRIPTION
**Pull request by AzureLatte**
## Description 📝
Some of our Cypress tests have stray calls to console.log and similar functions (console.info, console.error, etc.), and these should be removed.

## Changes  🔄
Remove console.log functions in the spec files (cypress/e2e/*/)

## How to test 🧪
```
yarn cy:run -s "cypress/e2e/core/longview/longview.spec.ts,cypress/e2e/core/images/create-linode-from-image.spec.ts"
```

## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

**Review by dwiley-akamai — COMMENTED** [review]
The diff LGTM and doing a search for `console.` in the codebase didn't show any other results in `cypress/e2e/*/`. The tests did fail for me locally though due a timeout so I'll approve once the remote tests pass

File: packages/manager/cypress/e2e/core/longview/longview.spec.ts [hunk 1]
```diff
@@ -154,15 +127,6 @@ describe('longview', () => {
           cy.findByText('Waiting for data...');
         });
```
Comment (jdamore-linode) [inline packages/manager/cypress/e2e/core/longview/longview.spec.ts:129]:
```suggestion
      // Install Longview on Linode by SSHing into machine and executing cURL command.
      installLongview(linodeIp, linodePassword, installCommand);
```
In this case we still want to call `installLongview`, we just don't want to log its output. (This is going to be moot soon thanks to M3-8107, but I'm not sure this test will pass reliably in the meantime without this)

Comment (AzureLatte) [reply packages/manager/cypress/e2e/core/longview/longview.spec.ts:156]: Thanks! I reverted the changes and put the ```installlongview()``` back.

**Review by jdamore-linode — APPROVED** [review]
Thanks, @AzureLatte!

**Review by dwiley-akamai — APPROVED** [review]

**Pull request merged**